### PR TITLE
MISUV-4575 : Prevent Negative Credit Amount Showing

### PR DESCRIPTION
**Pull request by AshokGitAccount**

### CODE REVIEW

File: app/models/financialDetails/BalanceDetails.scala [hunk 1]
```diff
@@ -31,6 +33,15 @@ case class BalanceDetails(balanceDueWithin30Days: BigDecimal,
   val total: Option[BigDecimal] = availableCredit.map{ total =>
     total - firstPendingAmountRequested.getOrElse(0) - secondPendingAmountRequested.getOrElse(0)
   }
+
```
Comment (avelx) [inline app/models/financialDetails/BalanceDetails.scala:36]: pls unit test this change

**Review by avelx — CHANGES_REQUESTED** [review]
pls add required unit tests

File: app/models/financialDetails/BalanceDetails.scala (hunk 1 above)
Comment (AshokGitAccount) [reply app/models/financialDetails/BalanceDetails.scala:36]: Added Unit tests

**Review by avelx — APPROVED** [review]
approved

File: test/models/financialDetails/BalanceDetailsSpec.scala [hunk 2]
```diff
@@ -95,5 +95,15 @@ class BalanceDetailsSpec extends UnitSpec {
         balanceDetailsModel().refundInProgress shouldBe false
       }
     }
+
+    "return absolute values" when {
```
Comment (avelx) [inline test/models/financialDetails/BalanceDetailsSpec.scala:99]: we would need a unit test for the case where is no value;

**Pull request merged**